src/components/CCPForm/CCPFormPage.tsx

```diff
@@ -112,24 +134,24 @@ const CCPFormPage = ({
     if (mode === 'edit') {
       editCCP({
         variables: {
-          id: ccpID,
+          id: ccpId,
           name: ccpName,
-          eventId: eventID,
+          eventId: eventId,
         },
       });
     } else {
       addCCP({
         variables: {
           name: ccpName,
-          eventId: eventID,
+          eventId: eventId,
           createdBy: 4,
```

Comment:
let's actually change this to 1 so that staging isn't broken (we only have a single user on staging right now)
```suggestion
          createdBy: 1,
```